eng/scripts/Track1CodeCheck.ps1

```diff
@@ -212,6 +212,8 @@ try {
             Write-Output $_
         }
         LogError "Generated code is manually altered, you may need to re-run sdk\<RP_Name>\generate.ps1"
```

Comment:
"Discrepancy detected between generated code in PR and reference generation. Please note, the files in the Generated folder should not be modified OR adding/excluding files. You may need to re-run sdk\<RP_Name>\generate.ps1.